platforms/hyperledger-indy/configuration/samples/network-indyv3-aries.yaml

```diff
@@ -96,13 +96,7 @@ network:
       # Services maps to the pods that will be deployed on the k8s cluster
       # This sample has trustee
       services:
-        trustees:
-        - trustee:
-          name: authority-trustee
-          genesis: true
-          server:
-            port: 8000
-            ambassador: 15010
+        trustee: authority-trustee
```

Comment:
only the list should be deleted. So it will look like
```yaml
services:
  trustee:
    name: xxx
    genesis: true
    server: ...
```

Reply:
Fixed.